diff --git a/cmd/derper/cert.go b/cmd/derper/cert.go
index db84aa5..adb971d 100644
--- a/cmd/derper/cert.go
+++ b/cmd/derper/cert.go
@@ -88,10 +88,6 @@ func (m *manualCertManager) TLSConfig() *tls.Config {
 }
 
 func (m *manualCertManager) getCertificate(hi *tls.ClientHelloInfo) (*tls.Certificate, error) {
-	if hi.ServerName != m.hostname {
-		return nil, fmt.Errorf("cert mismatch with hostname: %q", hi.ServerName)
-	}
-
 	// Return a shallow copy of the cert so the caller can append to its
 	// Certificate field.
 	certCopy := new(tls.Certificate)
